superset/connectors/sqla/models.py

```diff
@@ -226,8 +226,8 @@ class TableColumn(Model, BaseColumn, CertificationMixin):
     table_id = Column(Integer, ForeignKey("tables.id"))
     table: SqlaTable = relationship(
         "SqlaTable",
-        backref=backref("columns", cascade="all, delete-orphan"),
-        foreign_keys=[table_id],
+        back_populates="columns",
```

Comment:
`backrefs`, per [here](https://docs.sqlalchemy.org/en/14/orm/backref.html), is deemed legacy.